.github/workflows/nix.yml

```diff
@@ -7,11 +7,7 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - uses: cachix/install-nix-action@v12
-      with:
-        nix_path: nixpkgs=channel:nixos-unstable
     - uses: cachix/cachix-action@v8
       with:
         name: nix-community
-        extraPullNames: nixpkgs-update
```

Comment:
I was hoping to add this to avoid extra building work, but I'm fine if you don't want it.

Reply:
good point, restored